cddl/cri.cddl

```diff
@@ -1,12 +1,25 @@
 ; not expressed in this CDDL spec: trailing nulls to be left off
 
+RFC-XXXX-Definitions = [CRI, CRI-Reference]
```

Comment:
just a curiosity: why is this an array?

Reply:
It indeed could be anything; for random generation testing, it is best to simply generate one each.